src/Modules/Backend/templates/Backend/Actions/UserEdit.html.twig

```diff
@@ -48,5 +48,16 @@
     </div>
   </div>
 
+  {% if backupCodes %}
+    <div class="alert alert-warning">
+      {{ 'msg.BackupCodes'|trans|raw }}
+      <ul>
+        {% for code in backupCodes %}
+          <li>{{ code }}</li>
```

Comment:
Would it be an idea to add a copy-icon?
And show the codes in a `<code>` tag?

Reply:
Added copy to clipboard logic c2d11cf439aefbd19cfbc6aa4cea6c2e33ddd78b